reactMovieApp/src/components/movieComponents/MovieDescription.js

```diff
@@ -0,0 +1,19 @@
+import './MovieDescription.css'; 
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MovieDescription = (movie) => {
+    const info = movie.item;
+    
+    return (
+    <div className='text'>
+        <p>{info.overview}</p>
+    </div>
+    )
+}
```

Comment:
you can destructure movie.item and return inline
const MovieDescription = ({ item: info }) => (
    <div className='text'>
        <p>{info.overview}</p>
    </div>
)